platform/macos/export/export_plugin.cpp

```diff
@@ -165,17 +168,18 @@ void EditorExportPlatformMacOS::get_export_options(List<ExportOption> *r_options
 	r_options->push_back(ExportOption(PropertyInfo(Variant::PACKED_STRING_ARRAY, "codesign/custom_options"), PackedStringArray()));
 
 #ifdef MACOS_ENABLED
-	r_options->push_back(ExportOption(PropertyInfo(Variant::INT, "notarization/notarization", PROPERTY_HINT_ENUM, "Disabled,PyOxidizer rcodesign,Xcode altool"), 0, true));
+	r_options->push_back(ExportOption(PropertyInfo(Variant::INT, "notarization/notarization", PROPERTY_HINT_ENUM, "Disabled,PyOxidizer rcodesign,Xcode altool,Xcode notartytool"), 0, true));
```

Comment:
Should we include "(deprecated)" for altool?

Reply:
Added `(deprecated)`.

Reply:
Also, would changing the order / removing an entry break compatibility?

If eventually altool is no longer supported by Apple we'd want to remove it from the enum hint, and I guess this might silently change users to notarytool (which is not bad in practice but still maybe unexpected). And users on notarytool already might get an index out of range error for their existing presets?

BTW, typo at `notar*t*ytool`.

Reply:
This will change users to notary tool, but it should not be an issue, old login options (email, password) are the same.

Reply:
I have changed the order, keeping `altool` for now, since some people might be still using older Xcode.